src/framework/ui/internal/uiactionsregister.cpp

```diff
@@ -47,13 +47,19 @@ void UiActionsRegister::init()
 void UiActionsRegister::reg(const IUiActionsModulePtr& module)
 {
     const UiActionList& alist = module->actionsList();
+    ActionCodeList actionCodes;
     for (const UiAction& action : alist) {
         Info info;
         info.module = module;
         info.action = action;
         m_actions[action.code] = std::move(info);
+
+        actionCodes.push_back(action.code);
     }
 
+    updateEnabled(actionCodes);
```

Comment:
I don't quite understand the purpose of this. Could you please explain it to me?